docs/apps/interchain-accounts/overview.md

```diff
@@ -4,35 +4,28 @@ order: 1
 
 # Overview
 
-Learn about what the Interchain Accounts module is, and how to build custom modules that utilize Interchain Accounts functionality {synopsis}
-
+Learn about what the Interchain Accounts module is {synopsis}
```

Comment:
I've trimmed down the `Overview` doc quiet a bit. A lot of it had referenced authentication modules